### PR TITLE
[VIVO-1674] Improve quality of thumbnails created -- Maint rel 1.9 

### PR DESCRIPTION
**Pull request by delsborg**
**[JIRA Issue](https://jira.duraspace.org/browse/VIVO-1674)**: VIVO-1674

This pull request is for maint-rel-1.9

# What does this pull request do?
Address issue with quality of thumbnails created when uploading an image

# What's new?
Use Twelvemonkeys plugin for scaling thumbnails
Adds additional logic to the jpg creation to allow us to set jpg compression level
Adds in changes and dependencies from 1.10 into 1.9.3

# How should this be tested?
You can reproduce the issue by upload an image to VIVO. Observe the thumbnail that is created... it likely looks like it was cut out of a Norm's Diner menu in the 80s and somehow mimeographed onto the internet, i.e. not good.
image

Build Vitro/VIVO with the changes. Thumbnails should now be much better looking.
image

# Additional Notes:
The twelvemonkeys plugin for ImageIO is already being included in [Vitro dependencies](https://github.com/vivo-project/Vitro/blob/f2656ba3d8ae67359567d5aecd92eb67ba743c62/dependencies/pom.xml#L74)

As always, improvements are welcome.

# Interested parties
@gneissone @awoods @grahamtriggs

### CODE REVIEW

File: api/src/main/java/edu/cornell/mannlib/vitro/webapp/imageprocessor/imageio/IIOImageProcessor.java [hunk 1]
```diff
@@ -55,8 +54,8 @@
 	 */
 	@Override
 	public void startup(Application application, ComponentStartupStatus ss) {
-		JAI.getDefaultInstance().setImagingListener(
-				new NonNoisyImagingListener());
+//		JAI.getDefaultInstance().setImagingListener(
```
Comment (awoods) [inline api/src/main/java/edu/cornell/mannlib/vitro/webapp/imageprocessor/imageio/IIOImageProcessor.java]: Please remove, instead of commenting out.

Comment (delsborg) [reply api/src/main/java/edu/cornell/mannlib/vitro/webapp/imageprocessor/imageio/IIOImageProcessor.java]: done

**Comment by gneissone** [comment]
> One minor code review request.

Not sure why those lines were left in originally. I guess they are unneeded... https://github.com/vivo-project/Vitro/commit/45eef7e9f7e7ba040b9f316a73a4de2ffdd9c339#diff-acbe4ac1b341bae434f5781e370656c6R54

**Comment by delsborg** [comment]
I removed those lines from the maint-rel-1.9 branch, I imagine you want it removed on the develop branch too

**Pull request merged**